src/rookify/config.schema.yaml

```diff
@@ -24,6 +24,8 @@ rook:
     namespace: str()
   ceph:
     image: str()
+    public_network: regex("^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(/(3[0-2]|2[0-9]|1[0-9]|[0-9]))?$", required=False)
```

Comment:
I think this allows leading zeros, e.g. stuff like this: 192.168.001.001

will that be a problem?

Reply:
ok seems to be no problem :)